docs/release/release_0_4_19.md

```diff
@@ -0,0 +1,254 @@
+# napari 0.4.19
+
+We're happy to announce the release of napari 0.4.19!
+napari is a fast, interactive, multi-dimensional image viewer for Python.
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based
+rendering), and the scientific Python stack (numpy, scipy).
+
+
+For more information, examples, and documentation, please visit our website:
+https://github.com/napari/napari
```

Comment:
```suggestion
https://napari.org/stable/
```
?

Reply:
Thanks, I have update script for generation release notes.